test_data/makedata_suites/402_views.js

```diff
@@ -435,15 +908,15 @@ function deleteAnalyzer_400(testgroup, analyzerName){
         }
       }
 
-      testCases.forEach(test => {
+      arangosearchTestCases.forEach(test => {
         // create collection for each testing link
-        let collectionName = `${test["collectionName"]}_${loopCount}`;
+        let collectionName = `${test["collectionName"]}_as_${loopCount}`; // collections for testing 'arangosearch'
         createCollectionSafe(collectionName, 3, 1);
         // insert some test data. Also insert version, on which 'make_data' was called
         db._collection(collectionName).insert([
           { "animal": "cat", "name": "tom", "geo_location": { "type": "Point", "coordinates": [0.937, 50.932] }, "geo_latlng": { "lat": 50.932, "lng": 6.937 } },
-          { "animal": "mouse", "name": "jerry", "location": { "type": "Point", "coordinates": [12.7, 3.93] }, "geo_latlng": { "lat": 50.941, "lng": 6.956 } },
-          { "animal": "dog", "name": "harry", "location": { "type": "Point", "coordinates": [-6.27, 51.81] }, "geo_latlng": { "lat": 50.932, "lng": 6.962 } },
+          { "animal": "mouse", "name": "jerry", "geo_location": { "type": "Point", "coordinates": [12.7, 3.93] }, "geo_latlng": { "lat": 50.941, "lng": 6.956 } },
```

Comment:
does this mean we have an API-change?

Reply:
@dothebart 
No. I added wrong field name before. And now I changed it to correct one which I really want to index

Reply:
ok, tnx.